### ISSUE TITLE
[Document Manager] Failed to load fixtures

### ISSUE THREAD

**Issue by brieucthomas** [sulu/sulu#3458]
| Q | A
| --- | ---
| Bug? | yes
| New Feature? | no
| Sulu Version | ?
| Browser Version | 

#### Actual Behavior

Load data fixtures using the `sulu:document:fixtures:load` command failed with Symfony 3.x:

```bash
$ php app/console sulu:document:fixtures:load

[Symfony\Component\Console\Exception\InvalidArgumentException]
The helper "dialog" is not defined.
```

#### Expected Behavior

Load fixtures.

#### Steps to Reproduce

Documentation: http://docs.sulu.io/en/latest/reference/components/document-manager/fixtures.html

1. Use Symfony 3.x 
2. Create fixtures
3. Run the `sulu:document:fixtures:load` command

#### Possible Solutions

https://symfony.com/doc/current/components/console/helpers/dialoghelper.html

> The Dialog Helper was deprecated in Symfony 2.5 and removed in Symfony 3.0. **You should now use the Question Helper instead**, which is simpler to use.

Use the `Question Helper` when available.

```php
if ($this->getHelperSet()->has('question')) {
    $questionHelper = $this->getHelperSet()->get('question');
    // ...
} else {
    $dialog = $this->getHelperSet()->get('dialog');
    // ...         
}
```

**Comment by wachterjohannes** [comment]
@brieucthomas sorry for the delay. I have created a PR for sulu 1.6. thanks for reporting this issue

**Comment by alexander-schranz** [comment]
Fixed in #3470
